README.rst

```diff
@@ -54,13 +54,13 @@ An example workflow might be:
 
 * If the package contains FFV1 or Matroska files, perhaps ``ffv1mkvvalidate.py`` could run, which uses ``mediaconch`` to verify the compliance of the files, and stores the information in the logfile.
 
-* If the package passes our Quality Control Procedures, then it will be accessioned. ``accession.py`` will generate an accession number, rename the OE number with the accession number, generate a SHA-512 manifest and update the log file to document these new preservation events.
+* If the package passes our Quality Control Procedures, then it will be accessioned. ``aipcreator.py`` will generate an accession number, rename the OE number with the accession number, generate a SHA-512 manifest and update the log file to document these new preservation events.
 
-* A large batch of items can be accessioned using ``batchaccession.py``. If the ``-pbcore`` command line argument is used with the accessioning scripts, technical metadata based on the PBCore standard will be generated in CSV format. This process can be run seperately by using ``makepbcore.py``. CSV was chosen instead of XML as this allows us to immediately import the CSV into our database system so that we have item level records.
+* A large batch of items can be aipped using ``batchaipcreator.py``. If the ``-pbcore`` command line argument is used with the aipcreator scripts, technical metadata based on the PBCore standard will be generated in CSV format. This process can be run seperately by using ``makepbcore.py``. CSV was chosen instead of XML as this allows us to immediately import the CSV into our database system so that we have item level records.
```

Comment:
Perhaps “packaged as AIPs” instead of aipped?

Reply:
@hyz2993 suggest changing all remaining instances of 'aipped' to AIPed.

Reply:
Fixed.